webpack/assets/javascripts/dashboard/index.js

```diff
@@ -55,29 +59,33 @@ export function removeWidget(item) {
   const gridster = $('.gridster>ul')
     .gridster()
     .data('gridster');
-  if (
-    window.confirm(
-      __('Are you sure you want to delete this widget from your dashboard?')
-    )
-  ) {
-    $.ajax({
-      type: 'DELETE',
-      url: $(item).data('url'),
-      success() {
-        notify({
-          message: __('Widget removed from dashboard.'),
-          type: 'success',
-        });
-        gridster.remove_widget(widget);
-      },
-      error() {
-        notify({
-          message: __('Error removing widget from dashboard.'),
-          type: 'error',
-        });
-      },
-    });
-  }
+  openConfirmModal({
```

Comment:
You can test it now in the dashboard as well:

https://user-images.githubusercontent.com/26363699/124386171-cffabb00-dce1-11eb-8413-6320cc0117cc.mp4